packages/webapp/components/FooterNavBar.tsx

```diff
@@ -65,15 +65,15 @@ export default function FooterNavBar({
     'shadow-[0_4px_30px_rgba(0,0,0.1)]',
   );
 
-  const Component = isNewMobileLayout ? FooterNavBarV1 : FooterNavBarControl;
+  const Component = isLaptop ? FooterNavBarControl : FooterNavBarV1;
```

Comment:
We can completely remove the `FooterNavBarControl` and the component itself. This component is only available for tablet and mobile.

Reply:
👍 but why did we have even control in this case?

Reply:
If the user was under the old or new layout experiment.